packages/pirania/Makefile

```diff
@@ -13,12 +13,11 @@ define Package/$(PKG_NAME)
   CATEGORY:=Network
   MAINTAINER:=Asociación Civil AlterMundi <info@altermundi.net>
   TITLE:=Captive portal with vouchers.
-  DEPENDS:=+ip6tables-mod-nat +ipset +shared-state +shared-state-pirania \
+  DEPENDS:=+nftables +liblucihttp0 +uhttpd +uhttpd-mod-ubus +shared-state +shared-state-pirania \
```

Comment:
Can you rename the dependency `liblucihttp0` to `liblucihttp`? See the comment by jow- in the conversation here https://github.com/libremesh/lime-packages/pull/1164#issuecomment-2961499968 
You could even remove it, as it is pulled by liblucihttp-lua anyways.

Reply:
Thanks, done!